tests/all-tests.html

```diff
@@ -11,6 +11,11 @@
 
     <script>
     QUnit.testSuites("The Chart Authoring Tool Tests", [
+        "./html/d3ViewComponent-Tests.html",
+        "./html/pie-Tests.html",
+        "./html/legend-Tests.html",
+        "./html/pieChart-Tests.html",
+        "./html/d3Utils-Tests.html",
```

Comment:
The d3Utils-tests.html should come before the tests that make use of the d3Utils.js functions.